tla/consensus/ccfraft.tla

```diff
@@ -815,22 +817,22 @@ RejectAppendEntriesRequest(i, j, m, logOk) ==
        \/ /\ m.term >= currentTerm[i]
           /\ state[i] = Follower
           /\ ~logOk
-          /\ LET prevTerm == IF m.prevLogIndex = 0 THEN 0
-                             ELSE IF m.prevLogIndex > Len(log[i]) THEN 0 ELSE log[i][m.prevLogIndex].term
+          /\ LET prevTerm == IF m.prevLogIndex = 0 THEN 2
+                             ELSE IF m.prevLogIndex > Len(log[i]) THEN 2 ELSE log[i][m.prevLogIndex].term
              IN /\ m.prevLogTerm # prevTerm
-                /\ \/ /\ prevTerm = 0
+                /\ \/ /\ prevTerm = 2
                       /\ Reply([type        |-> AppendEntriesResponse,
                              success        |-> FALSE,
                              term           |-> currentTerm[i],
                              lastLogIndex   |-> Len(log[i]),
                              source         |-> i,
                              dest           |-> j],
                              m)
-                   \/ /\ prevTerm # 0
+                   \/ /\ prevTerm # 2
                       /\ LET lli == FindHighestPossibleMatch(log[i], m.prevLogIndex, m.term)
                          IN Reply([type        |-> AppendEntriesResponse,
                                 success        |-> FALSE,
-                                term           |-> IF lli = 0 THEN 0 ELSE log[i][lli].term,
+                                term           |-> IF lli = 0 THEN 2 ELSE log[i][lli].term,
```

Comment:
Check this in the implementation.

Reply:
@heidihoward the implementation does in fact send the term of the highest possible on rejected append entries https://github.com/microsoft/CCF/blob/1d52533f70a94d015f3215acec97d629e11bfdd7/src/consensus/aft/raft.h#L1351, instead of the current term, and this used when deciding what to resend: https://github.com/microsoft/CCF/blob/1d52533f70a94d015f3215acec97d629e11bfdd7/src/consensus/aft/raft.h#L1473